resources/assets/components/CampaignIdSingle/index.js

```diff
@@ -83,18 +83,12 @@ class CampaignIdSingle extends React.Component {
           )}
           <h4>Start Date</h4>
           <p>
-            {format(
-              new Date(campaign.start_date.replace(/-/g, '/')),
-              'MM/dd/yyyy',
-            )}
+            {format(new Date(campaign.start_date.replace(/-/g, '/')), 'M/d/yy')}
```

Comment:
It might be cleaner to use `date-fn`'s [`parse`](https://date-fns.org/v1.30.1/docs/parse) method here (which I believe should support the given format), rather than applying a regex find-replace to "hack" things into the format that JavaScript is expecting.

Reply:
@DFurnes I thought about using that but think it would get a little messy too trying to format the dates since they are `DATETIME`s and `parse` expects a `TIMESTAMP`?

Reply:
Where are you seeing that `parse` expects a `TIMESTAMP`?

Reply:
Oh, I totally read that wrong! I'm still having a hard time getting this to work though, still seeing "Invalid Date" when doing something like: `{parse(campaign.updated_at, 'd.MM.yy h:m:s', new Date()).toString()}`

Reply:
I'm not sure about the usage of `parse()` with three parameters and `.toString()` there. Can you link to where in the docs you're seeing that?

This seems to work for me:

```js
format(parse(campaign.updated_at), 'M/d/YY');
```

Reply:
hm that still doesn't work for me when I test out in Safari :-/ 

I'm getting this error: `TypeError: 3 arguments required, but only 1 present` from `parse`. 

I found the example [here](https://date-fns.org/v2.0.0-alpha.27/docs/parse#v2.0.0-breaking-changes) to use three parameters after seeing this error (and even tried to drop the `.toString()` that I found [here](https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md) but doesn't seem to be working :-/ 

What is weird though is that I thought we were using `v1.30.1` but it seems to still be adopting the new rules...

Reply:
Huh – yeah, it looks like we [ended up with the 2.0 beta](https://github.com/DoSomething/rogue/blob/ec3eb8a9629a87b2e18b5adf351dd16476296ce1/package-lock.json#L2918-L2922) somehow.

Reply:
Not sure why that's happening, but looks like you can use [`toDate`](https://date-fns.org/v2.0.0-alpha.27/docs/toDate) for this in 2.x:

```js
format(toDate("2019-03-28"), 'M/d/YY');
```

Reply:
@DFurnes updated to the correct version and all is working now! Thanks for the help on that! PR [here](https://github.com/DoSomething/rogue/pull/857).